editor-service/src/main/java/com/qreal/wmp/editor/database/exceptions/translation/ExceptionTranslationClients.java

```diff
@@ -22,7 +22,7 @@
 @EnableAspectJAutoProxy(proxyTargetClass = true)
 public class ExceptionTranslationClients {
```

Comment:
We need here one more exception translation function for palette service (do not changed alredy existing).